src/interactivity/feature.js

```diff
@@ -21,7 +21,7 @@ export default class Feature {
             variables[varName] = new FeatureVizProperty(`__cartovl_variable_${varName}`, rawFeature, viz, customizedFeatures, trackFeatureViz, idProperty);
         });
 
-        this.id = rawFeature.id;
+        this.id = rawFeature[idProperty];
```

Comment:
Could we name it `propertyName` instead of `idProperty` to be consistent with https://github.com/CartoDB/carto-vl/pull/736/files#diff-3ef34d2cff5a70a4c3874360d727d466R281 and https://github.com/CartoDB/carto-vl/pull/736/files#diff-015175475be99f8a24e412bd4790db19R16?

Reply:
After talking with @davidmanzanares this is actually the name used as id, so the name is ok 👌